R/USairports.R

```diff
@@ -43,9 +43,9 @@
 #'     \item{Distance}{The distance between the two airports, in miles.}
 #'   }
 #' @source
-#'   Most of this information was downloaded from The Research and
+#'   Most of this information was downloaded from The (now dissolved) Research and
 #'   Innovative Technology Administration (RITA). See
-#'   <http://www.rita.dot.gov/about_rita/> for details. The airport
```

Comment:
@szhorvat a big issue is that CRAN will flag broken link even if they're not formatted as links.